app/assemblers/assemble_exercise_representations_with_feedback.rb

```diff
@@ -0,0 +1,37 @@
+class AssembleExerciseRepresentationsWithFeedback
+  include Mandate
+
+  def self.keys = %i[page order criteria track_slug]
+
+  initialize_with :mentor, :params
+
+  def call
+    SerializePaginatedCollection.(
+      representations,
+      serializer: SerializeExerciseRepresentations,
+      serializer_kwargs: { with_feedback: false },
+      meta: {
+        unscoped_total: Exercise::Representation.with_feedback.count
```

Comment:
See comment below about caching. Less important on this one, but maybe still worth doing.